packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -129,6 +170,12 @@ To enable grouped batching:
     }
 ```
 
+## Note about performance and latency
+
+In terms of performance and impact on latency, the results greatly depend on payload size, payload structure, network speed and CPU speed. Therefore, customers must perform the required measurements and adjust the settings according to their scenarios.
+
+In general, compression offers a trade-off between higher compute costs, lower bandwidth consumption and lower storage requirements, while chunking slightly increases latency due to the overhead of splitting an op, sending the chunks and reconstructing them on each client. Grouped batching heavily decreases the number of ops observed by the server and slightly decreases the bandwidth requirements as it merges all the ops in a batch into a single op and also eliminates the op envelope overhead.
```

Comment:
I'd probably not give user a choice to disable chunking, and thus not mention it here - there are no benefits for users here (broken system has no perf metrics :))
I think grouping is going to be either always positive, or non-measuable (in terms of regression). Do we have any reasons to believe users may benefit from disabling it? If not, we should probably state that it's neutral and not give option to disable.

Compression is the only thing that is interesting in terms of giving knobs to users I think.

The less options we give to users - the better :)

Reply:
Not sure which part you're referring to (either the example configs or the feature flags), but I will remove some configs, specifically the ones which enable compression and not chunking.

As for grouped batching, we do still consider it experimental and opt-in with code changes. Disabling it without code changes (via feature flags) should be documented as well.